amazonredshift_blueprints/upload_file.py

```diff
@@ -76,21 +76,10 @@ def get_args():
     return args
 
 
-def create_connection_url(host:str, password:str, user:str, database:str,port = 5439):
-    url = URL.create(drivername= 'redshift+redshift_connector', host = host, password= password, username= user, port = port, database=database)
+def create_connection_url(host: str, password: str, user: str, database: str, port=5439):
+    url = URL.create(drivername='redshift+redshift_connector', host=host, password=password, username=user, port=port,
+                     database=database)
     return url
-def create_connection_string(args):
```

Comment:
I couldn't see anywhere where this was being used. I imagine this was used at one point but wasn't cleaned up

Reply:
Yep this was a legacy function, fine to remove